cloud/blockstore/libs/storage/protos/part.proto

```diff
@@ -73,6 +73,8 @@ message TPartitionConfig
 
     // Optional tablet id of base volume.
     uint64 BaseDiskTabletId = 17;
+
+    bool IsProxyOverlay = 18;
```

Comment:
"ProxyOverlay" is not Blockstore concept.
Maybe it is better to disable logical used blocks calculation when `IsSystem == true` and `!BaseDiskId.empty()`?
Or even when `IsSystem == true`.

And this fix looks like "kostyl", so write comment about that, please

Reply:
i dont know that proxy overlay disks is system

fixed